rasa_nlu/data_router.py

```diff
@@ -346,3 +346,20 @@ def evaluate(self, data, project=None, model=None):
                 "f1_score": f1,
                 "accuracy": accuracy}
         }
+
+    def unload_model(self, project, model):
+        # type: (Text, Text) -> Dict[Text]
+        """Unload a model from server memory."""
+
+        project = project or RasaNLUConfig.DEFAULT_PROJECT_NAME
+
+        if project not in self.project_store:
+            raise InvalidProjectError("Project {} could not "
+                                      "be found".format(project))
+
+        try:
+            unloaded_model = self.project_store[project].unload(model)
+            return {"unloaded_model": unloaded_model}
```

Comment:
I think it is fine to just return the model without the `"unloaded_model"` wrapper, so just `return unloaded_model`